dynawo/CMakeLists.txt

```diff
@@ -113,6 +113,10 @@ elseif('${CMAKE_CXX_COMPILER_ID}' STREQUAL 'Clang')
   set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Wno-return-std-move-in-c++11")
 endif()
 
+if(CMAKE_BUILD_TYPE STREQUAL "Debug" OR RELEASE_WITH_DEBUG STREQUAL "YES")
+  add_compile_definitions(_DEBUG_)
```

Comment:
You can't do this here you need to do it at the same place than before

Reply:
I wonder why the _DEBUG_ macro was defined here while it's not used exclusively in its original folder dynawo/sources/Common

Reply:
Same place = the dynawo_Interface target. In fact this target could be moved from Common to the main CMakeLists